vars/cloudFoundryDeploy.groovy

```diff
@@ -196,11 +196,20 @@ private String deleteOptionIfRequired(Map config) {
     }
 }
 
-private String stopOldAppIfRequired(Map config) {
+private void stopOldAppIfRequired(Map config) {
+    String oldAppName = "${config.cloudFoundry.appName}-old"
```

Comment:
By the way. What if config.cloudFoundry.appName is null? I think this is possible, right?